containers/ruby-consul-terraform/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM broadinstitute/dsde-toolbox:consul-0.20.0
```

Comment:
Docker container for running terraform that also has consul-template 20 for rendering and gsutil for the sync